src/main/resources/com/cloudbees/plugins/credentials/CredentialsStoreAction/DomainWrapper/index.jelly

```diff
@@ -23,98 +23,131 @@
  ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  ~ THE SOFTWARE.
  -->
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout"
-         xmlns:f="/lib/form" xmlns:c="/lib/credentials"  xmlns:t="/lib/hudson">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:dd="/lib/layout/dropdowns"
+         xmlns:f="/lib/form" xmlns:c="/lib/credentials" xmlns:t="/lib/hudson">
   <l:layout title="${it.fullDisplayName}" permission="${it.parent.VIEW}" type="one-column">
+    <st:adjunct includes="com.cloudbees.plugins.credentials.common.card"/>
+
+    <j:set var="creds" value="${it.credentials.values()}"/>
     <l:main-panel>
+      <l:dialog title="${%Add Credentials}" hash="new-credentials">
```

Comment:
Should the dialog include the domain name that has been selected?

<img width="598" height="235" alt="image" src="https://github.com/user-attachments/assets/448de7ef-986b-4e64-99cf-9413b2360332" />